src/main/java/net/gigaclub/buildersystemplugin/cache/WorldCache.java

```diff
@@ -145,25 +152,26 @@ public void worldinfo(int iworld, Inventory inventory, int i, JSONArray worlds)
         worldlore.add(ChatColor.WHITE + " " );
         worldlore.add(ChatColor.WHITE + "builder_team.world.list.world_manager_user" );
 
-        JSONArray users = world.getJSONArray("user_manager_ids");
+        JSONArray users = world.getJSONArray("user_ids");
         for (int j = 0; j < users.length(); j++) {
 
             JSONObject user = users.getJSONObject(j);
 
-            Player player11 = Bukkit.getPlayer(user.getString("name"));
-            String player21 = player11.toString();
+            UUID uuid = UUID.fromString(user.getString("mc_uuid"));
+            OfflinePlayer player11 = Bukkit.getOfflinePlayer(uuid);
+            String player21 = player11.getName();
             StringBuilder res1 = new StringBuilder();
```

Comment:
@Feier68 do not use numbers in variables. If you have duplicate variable names, assign the variable above and name the variables correctly.